src/docs/components/styling.md

```diff
@@ -162,6 +162,45 @@ shadow-card::part(heading) {
 
 This allows for greater flexibility in styling as any styles can now be added to this element.
 
+### Exportparts
+
+If you have a Stencil component nested within another component, any `part` specified on elements of the child component will not be exposed through the parent component. In order to expose the `part`s of the child component, you need to use the `exportparts` attribute. Consider this `OuterComponent` which contains the `InnerComponent`.
+
+```tsx
+export class OuterComponent {
+  render() {
+    return (
+      <Host>
+        <h1>Outer Component</h1>
+        <inner-component exportparts="inner-text" />
+      </Host>
+    );
+  }
+}
+
+export class InnerComponent {
```

Comment:
It may be worth adding a `@Component` decorator to this class, so that we define a tag to be associated with it. That way, [this line](https://github.com/ionic-team/stencil-site/pull/830/files#diff-fdd264bad2c75226b929f996da39e4dbee5ced97ea40f7c2eeb86ef9ea110650R175) makes a little more sense. (also consider adding `shadow` as well)

Perhaps we should add it (`@Component`) for both, since we reference `outer-component` [here](https://github.com/ionic-team/stencil-site/pull/830/files#diff-fdd264bad2c75226b929f996da39e4dbee5ced97ea40f7c2eeb86ef9ea110650R201)